ui/app/styles/utils/_color_variables.scss

```diff
@@ -3,23 +3,6 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-// Colors that will eventually be in Structure package
-
-// Vault Gray
-$vault-gray-010: #fdfdfe;
-$vault-gray-050: #f7fafc;
-$vault-gray-100: #ebeef5;
-$vault-gray-200: #d3dbe6;
-$vault-gray-300: #b4bdcc;
-$vault-gray-400: #949fb0;
-$vault-gray-500: #7c8797;
-$vault-gray-600: #5a6370;
-$vault-gray-700: #3c434d;
-$vault-gray-800: #1d2126;
-$vault-gray-900: #060708;
-$vault-gray: $vault-gray-500;
-$vault-gray-dark: $vault-gray-700;
```

Comment:
I always wondered why there was `vault-gray` and `ui-gray` variables. I take it all of the instance of `vault-gray` have been replace with other vars?

Reply:
Yeah, I wondered the same thing when first starting with Vault. I think a while ago Chelsea and I came to the conclusion that ui-gray was the more recently added and more consistently used so to stick with those colors. It felt sooo good to remove this chunk of vars.